configure.ac

```diff
@@ -2358,8 +2359,16 @@ AC_ARG_WITH([libdpdk],
 )
 
 if test "x$with_libdpdk" != "xno"; then
+  PKG_CHECK_MODULES(DPDK, [libdpdk],
+    [LIBDPDK_LIBS="$DPDK_LIBS"],
```

Comment:
I thought for the new var ther could be no value yet, but you are right.
Even if it would not have overwritten something your question proves it lacks readability.

So I modified it to follow the common pattern of if-not-set -> then if pkg-config use that; otherwise set default. This leaves any value set before in place to be sure (The same way it was done for cflags already).